index.js

```diff
@@ -64,8 +65,24 @@ HtmlWebpackPlugin.prototype.apply = function (compiler) {
 
   compiler.plugin('emit', function (compilation, callback) {
     var applyPluginsAsyncWaterfall = self.applyPluginsAsyncWaterfall(compilation);
-    // Get all chunks
-    var allChunks = compilation.getStats().toJson().chunks;
+    // Get chunks info as json
+    // Note: we're excluding stuff that we don't need to improve toJson serialization speed.
+    var chunkOnlyConfig = {
+      assets: false,
+      cached: false,
+      children: false,
+      chunks: true,
+      chunkModules: false,
+      chunkOrigins: false,
+      errorDetails: false,
+      hash: false,
+      modules: false,
+      reasons: false,
+      source: false,
+      timings: false,
+      version: false
+    };
```

Comment:
I think this could be simplified according to the [docs](https://webpack.js.org/configuration/stats/)

```javascript
var chunkOnlyConfig = {
  all: false,
  chunks: true
}
```

Reply:
Cool good point! 👍 
We will remove the getStats call in #953

Reply:
Even better 👍. Thanks for your work!